themes/godotengine/pages/governance.htm

```diff
@@ -0,0 +1,75 @@
+title = "Governance"
+url = "/governance"
+layout = "more"
+description = "Governance model"
+is_hidden = 0
+==
+{##}
+{% put title %} Governance {% endput %}
+
+<div class="container">
+
+  <h3>Legal status</h3>
+  <p>
+    Godot is legally a part of the
+    <a href="https://sfconservancy.org">Software Freedom Conservancy</a>. As a
+    not-for profit charity, the Software Freedom Conservancy receives donations
+    and, based on their mission statement, uses them for the benefit of the
+    project.
+    <strong>As no company is behind Godot, it can't be sold or purchased.</strong>
+  </p>
+  <p>
+    Within the organization sits the Godot
+    <a href="/contact#plc">Project Leadership Comittee</a>. This committee
+    decides and authorizes the usage of project funds (donations and grants).
+  </p>
+  <p>
+    <a href="/license">Godot's license is MIT</a>. This means you can do pretty
+    much everything with it, as long as you don't misrepresent the origin of the
+    software. In practice, even with that condition in place, you can do
+    anything you want as if you wrote Godot yourself.
+  </p>
+  <p>
+    Still, the MIT license allows creating proprietary derivatives.
+    Many companies have private forks of the engine to suit their own needs, but
+    what is open will remain open forever. <strong>No legal way exists to revoke
+    your right to use Godot.</strong> It's as if it was your own property;
+    nobody can take it from you.
+  </p>
+  <p>
+    Godot is made by thousands of skilled contributors who donate their free
+    time (some of them are paid from donations). Even if some contributors
+    suddenly decided to push new Godot versions as proprietary, it would be
+    difficult for them to compete or be as productive as the rest. Godot is a
+    true community effort, created and owned by everyone.
+  </p>
+
+  <h3>Project leadership, fund usage and decisions</h3>
+  <p>
+    While Godot has a <a href="/contact#plc">Project Leadership Committee</a>,
+    <strong>there is no central authority deciding what gets done</strong>.
+    The Project Leadership Committee rarely decides something on its own.
+    Usage of funds is discussed and approved by the
+    <em>advisors</em>, which is a group of about 30 veteran trusted contributors
+    and area maintainers. We hold online meetings and discuss these topics
+    regularly.
+  </p>
+  <p>
+    Godot is a large project and the advisors have a great reach to all
+    areas of the project, both technical and social. Our meetings are generally
+    well-informed, which allows steering the project direction more precisely.
+  </p>
+  <p>
+    Contributors cannot request to join the advisors or the Project Leadership
+    Committee. Instead, contributions in Godot are based on merit. Contributors
```

Comment:
> Instead, contributions in Godot are based on merit.

This is a bit confusing, what are you trying to communicate?

Reply:
The sentence doesn't make much sense on its own, so I'll remove it.